torch/onnx/symbolic_opset12.py

```diff
@@ -209,3 +211,58 @@ def unfold(g, input, dimension, size, step):
         return squeeze
     else:
         return _unimplemented("Unfold", "input size not accessible")
+
+@parse_args('v', 'v', 'is', 'is', 'v')
+def tensordot(g, input_a, input_b, dims_a, dims_b, out=None):
+    if out is not None:
+        _unimplemented("Tensordot", "Out parameter is not supported for tensordot.")
+
+    dim_count_a = sym_help._get_tensor_rank(input_a)
+    if dim_count_a is None:
+        raise RuntimeError('Unsupported: ONNX export of tensordot for tensor(input_a) of unknown rank.')
+
+    dim_count_b = sym_help._get_tensor_rank(input_b)
+    if dim_count_b is None:
+        raise RuntimeError('Unsupported: ONNX export of tensordot for tensor(input_b) of unknown rank.')
+
+    dims_a = [(dims_a[i] + dim_count_a)
```

Comment:
nit: `dims_a = [dim + dim_count_a if dim < 0 else dim for dim in dims_a]`

Reply:
Done.